junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/JavaTimeConversionPattern.java

```diff
@@ -45,4 +45,15 @@
 	 */
 	String value();
 
+	/**
+	 * Whether {@code null} argument values are allowed.
+	 *
+	 * <p>Defaults to {@code false}, in which case a {@code null} value will result in
+	 * an exception.
+	 *
+	 * @since 5.12
+	 */
+	@API(status = STABLE, since = "5.12")
```

Comment:
```suggestion
	@API(status = EXPERIMENTAL, since = "5.12")
```

When we initially introduce a feature, we typically assign it the `EXPERIMENTAL` status and then switch to `STABLE` in the next minor release if there are no issues with the feature.

Reply:
Makes sense 👍